tests/test_urls_only.py

```diff
@@ -0,0 +1,144 @@
+"""Test the URL parsing against the urls here: https://mathiasbynens.be/demo/url-regex."""
```

Comment:
This test was copied from https://github.com/fhightower/ioc-finder/blob/9c003b3aae25cf9751103e1f8a095b05e4cda059/tests/test_urls.py, correct?

Rather than add a new file, let's just add some test functions testing the `data_types` functionality to [this file](https://github.com/fhightower/ioc-finder/blob/main/tests/test_ioc_finder.py).